src/test/java/HomeWork/pages/DynamicControlsPage.java

```diff
@@ -38,7 +41,20 @@ public void clickInputButton() {
         driver.findElement(INPUT_BUTTON).click();
     }
 
-    public boolean isInputDisabled(WebDriver driver, By by) {
-        return driver.findElement(by).getAttribute("disabled") != null;
+    public boolean isInputAvailable(By by) {
+        return driver.findElement(by).isEnabled();
     }
+
+    public By getCHECK_CHECKBOX() {
```

Comment:
Why do you need these public methods which return By locator here? All locators should be incapsulated in page classes and you should have only public methods here that perform actions on these elements like click, getText, isEnabled...